CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Next
 
+* [#823](https://github.com/DestinyItemManager/DIM/issues/823) Added 'current' property to stores.
```

Comment:
I've been avoiding including "dev-only" stuff in the changelog - do you want to start?